### ISSUE TITLE
Timeout when translating with OpenAI connector

### ISSUE THREAD

**Issue by FransdeJong** [Jumoo/Jumoo.TranslationManager.Issues#61]
I'm running translationmanager 13.1.3 and OpenAi 1.0.1

When running a bigger task I get a empty error:
![image](https://github.com/user-attachments/assets/e547fa00-4ee9-4bd3-9eab-a91982a59f77)

When opening the console I see the following error:
![image](https://github.com/user-attachments/assets/3b364654-ccff-4dea-9788-0db8b54390a5)

Sometimes the job is created despite of the error and other times there is no job created.

**Comment by KevinJump** [comment]
Yes, i guess you are hitting a timeout in the requests to the webserver (default might be 100 seconds, azure limit is 240 seconds) . 

We do have mitigation for this in lots of parts of Translation manager, but not in the connector's submission methods - usually these are generating files, firing, and forgetting - but for machine translation they are a little slower (especially OpenAI).

if you can up the time, i would recommend it. If you can't at the moment the 'solution' is to send smaller blocks, see how low you can turn down the throttle in the connector settings.

**Comment by FransdeJong** [comment]
It seems in the background translationmanager is still running. I found the complete site translation after an hour in the translationsection. So it's only the front-end timing out?

If so this could be clarified in a error message. Something like:
_The request timed out. Please check the translation task in an hour or send smaller blocks for translation_?
